Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -190,107 +175,39 @@ final class RewardsCollectionViewController: UICollectionViewController {
-      from: layout,
-      using: self.collectionView
-    )
-    let needsUpdate = self.collectionView.contentInset.left != contentInsetLeftRight
-      || self.hiddenPagingScrollView.contentSize != contentSize
-
-    // Check if orientation or frame has changed
-    guard needsUpdate else {
-      return
-    }
-
-    _ = self.hiddenPagingScrollView
-      |> \.frame .~ self.collectionView.frame
-      |> \.bounds .~ CGRect(x: 0, y: 0, width: pageSize.width, height: pageSize.height)
-      |> \.contentSize .~ CGSize(width: contentSize.width, height: contentSize.height)
-
-    let (top, bottom) = self.collectionView.contentInset.topBottom
-
-    _ = self.collectionView
-      |> \.contentInset .~ .init(
-        top: top,
-        left: contentInsetLeftRight,
-        bottom: bottom,
-        right: contentInsetLeftRight
-      )
-
-    self.collectionView.contentOffset.x = -contentInsetLeftRight
-  }
-
-  private typealias HiddenScrollViewData = (
-    contentSize: CGSize, pageSize: CGSize,
-    contentInsetLeftRight: CGFloat
-  )
-
-  private func hiddenScrollViewData(
-    from layout: UICollectionViewFlowLayout,
-    using collectionView: UICollectionView
-  ) -> HiddenScrollViewData {
-    let itemSize = layout.itemSize
-    let lineSpacing = layout.minimumLineSpacing
-    let totalItemWidth = itemSize.width + lineSpacing
-
-    let pageWidth = totalItemWidth
-    let pageHeight = itemSize.height
-    let pageSize = CGSize(width: pageWidth, height: pageHeight)
-
-    let contentSize = CGSize(
-      width: collectionView.contentSize.width + lineSpacing,
-      height: collectionView.contentSize.height
-    )
-
-    let contentInsetLeftRight = (collectionView.frame.width - itemSize.width) / 2
-
-    return (contentSize, pageSize, contentInsetLeftRight)
-  }
-
   private func calculateItemSize(
     from layout: UICollectionViewFlowLayout,
     using collectionView: UICollectionView
   ) -> CGSize {
-    let cardWidth = CheckoutConstants.RewardCard.Layout.width
-
     let sectionInsets = layout.sectionInset
     let adjustedContentInset = collectionView.adjustedContentInset
 
     let topBottomSectionInsets = sectionInsets.top + sectionInsets.bottom
     let topBottomContentInsets = adjustedContentInset.top + adjustedContentInset.bottom
-    let leftRightInsets = sectionInsets.left + sectionInsets.right
 
     let itemHeight = collectionView.frame.height - topBottomSectionInsets - topBottomContentInsets
-    let itemWidth = cardWidth - leftRightInsets
+    let itemWidth = CheckoutConstants.RewardCard.Layout.width
 
     return CGSize(width: itemWidth, height: itemHeight)
   }
 
+  private func configureRewardsCollectionViewFooter(with count: Int) {
+    self.rewardsCollectionFooterView.configure(with: count)
+  }
+
+  private func updateRewardCollectionViewFooterConstraints(_ isHidden: Bool) {
+    self.collectionViewBottomConstraintSuperview?.isActive = false
```

Comment:
What is the reason for first disabling the constraint and then enabling or disabling it based on the `isHidden` boolean? Aren't these two lines `L:199` & `L:200` obsolete?

Reply:
This accidentally got in from some testing I was doing. Removed.